man/systemd.network.xml

```diff
@@ -964,6 +964,34 @@
           </listitem>
         </varlistentry>
         <varlistentry>
+          <term><varname>Anonymize=</varname></term>
+          <listitem>
+            <para>When true, the options sent to the DHCP server will
+            follow the RFC 7844 (Anonymity Profiles) to minimize disclosure
+            of identifying information.
+            Defaults to false.</para>
+
+            <para>This option should only be set to true when
+            <option>MACAddressPolicy=</option> is set to <literal>random</literal>
+            (see <citerefentry
+            project='man-pages'><refentrytitle>systemd.link</refentrytitle><manvolnum>5</manvolnum></citerefentry>).</para>
+
+            <para>Note that this configuration will take precedence when it is
+            set after others. In concrete, the following variables will be
+            ignored when set before:
+            <option>SendHostname=</option>, <option>ClientIdentifier=</option>,
+            <option>UseRoutes=</option>, <option>SendHostname=</option>,
+            <option>UseMTU=</option>, <option>VendorClassIdentifier=</option>,
+            <option>UseTimezone=</option>.</para>
+
+            <para>However, if any of those variables is set after, they will
+            override Anonymize. The client would have then a customized
```

Comment:
please write this as `<varname>Anonymize=</varname>`, we try to stick to the same formatting for all settings